polygeist/tools/cgeist/Lib/CodeGenTypes.cc

```diff
@@ -1347,7 +1347,8 @@ mlir::Type CodeGenTypes::getMLIRType(clang::QualType QT, bool *ImplicitRef,
           TypeName == "nd_range" || TypeName == "range" ||
           TypeName == "sub_group" ||
           TypeName == "TupleCopyAssignableValueHolder" ||
-          TypeName == "TupleValueHolder" || TypeName == "vec") {
+          TypeName == "TupleValueHolder" || TypeName == "vec" ||
+          TypeName == "multi_ptr") {
```

Comment:
can you please put it in alphabetical order?

Reply:
Sure